web-app/preprocess/csvToJson.py

```diff
@@ -0,0 +1,19 @@
+import csv
```

Comment:
maybe add a header comment describing the purpose of this script and how to run it and when to run it?